addons/web/static/src/legacy/xml/base.xml

```diff
@@ -1336,6 +1336,48 @@
         <span><img t-attf-src="/web/image/#{avatarModel}/#{el.id}/#{avatarField}" t-att-data-id="el.id" class="rounded-circle o_m2m_avatar"/></span>
     </t>
 </t>
+<t t-name="KanbanMany2ManyTagAvatar">
+    <t t-set="visibleElements" t-value="elements.length === widget.visibleAvatarCount ? elements : elements.slice(0, widget.visibleAvatarCount - 1)"/>
+    <t t-foreach="visibleElements" t-as="el">
+        <span><img t-attf-src="/web/image/#{avatarModel}/#{el.id}/#{avatarField}" t-att-title="el.display_name" t-att-data-id="el.id" class="rounded-circle o_m2m_avatar"/></span>
+    </t>
+    <t t-if="elements.length > widget.visibleAvatarCount">
+        <span class="o_m2m_avatar_empty rounded-circle text-center font-weight-bold">
+            <span t-if="elements.length > 9">9+</span>
```

Comment:
@msh-odoo That's wrong, should be more like the line below:
```
elements.length - widget.visibleAvatarCount + 1 > 9
```

Reply:
@SimonGenin Done